src/js/views/createDevice/steps/SecurityStep/SecurityTable.jsx

```diff
@@ -0,0 +1,141 @@
+import React, { useMemo } from 'react';
+
+import {
+  Radio,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableRow,
+  CircularProgress,
+  Typography,
+  Box,
+} from '@material-ui/core';
+import { DataTableHead } from 'Components/DataTable';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+
+import Pagination from './Pagination';
+import { useSecurityTableStyles } from './style';
+
+const SecurityTable = ({
+  page,
+  certificates,
+  totalPages,
+  rowsPerPage,
+  selectedCertificate,
+  isLoading,
+  handleChangePage,
+  setSelectedCertificate,
+  handleChangeRowsPerPage,
+}) => {
+  const { t } = useTranslation('createDevice');
+  const classes = useSecurityTableStyles();
+  const headCells = useMemo(
+    () => [
+      {
+        id: 'certificate',
+        label: t('securityStep.certificateName'),
+      },
+      {
+        id: 'creationTime',
+        label: t('securityStep.creationTime'),
+      },
+      {
+        id: 'expirationTime',
+        label: t('securityStep.expirationTime'),
+      },
+    ],
+    [t],
+  );
+
+  const handleCertificateSelection = cert => {
+    setSelectedCertificate(cert.id);
+  };
+
+  if (isLoading) {
+    return (
+      <Box className={classes.loadingContainer} padding={2}>
+        <CircularProgress size={24} />
+      </Box>
+    );
+  }
+
+  return (
+    <>
+      <TableContainer>
+        <Table aria-labelledby='tableTitle'>
+          <DataTableHead
+            className={classes.tableHead}
+            cells={headCells}
+            rowCount={certificates.length}
+            disableCheckbox
+            disableOrderBy
+          />
+
+          <TableBody>
+            {certificates.slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage).map(cert => {
+              const isSelected = selectedCertificate === cert.id;
+
+              return (
+                <TableRow
+                  key={cert.id}
+                  tabIndex={-1}
+                  role='radio'
+                  onClick={() => handleCertificateSelection(cert)}
+                  hover
+                >
+                  <TableCell padding='checkbox'>
+                    <Radio
+                      color='primary'
+                      checked={isSelected}
+                      onChange={() => handleCertificateSelection(cert)}
+                    />
+                  </TableCell>
+
+                  <TableCell className={classes.clickableCell}>Nome do Certificado</TableCell>
```

Comment:
Maybe you can use the certificate attributes here, example: `cert.label`, `cert.created`, `cert.expiration`. You can also use moment to format dates.